httpd/handler.go

```diff
@@ -175,6 +179,127 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 	httpResults(w, results, pretty)
 }
 
+type Point struct {
```

Comment:
I think we can still use the `client.Point` and `client.BatchPoints` for this.  That being said, I'm a little torn on if we should do it.  I think this comes back to me still wanting to separate some of these `models` to their own package for better re-use.  This comment is more food for thought at this point than an action item.